javascript/src/main/pages/SearchHistory/Alluser.js

```diff
@@ -0,0 +1,33 @@
+import React from "react";
+import BootstrapTable from 'react-bootstrap-table-next';
+import useSWR from "swr";
+import {useAuth0} from "@auth0/auth0-react";
+import {fetchWithToken} from "main/utils/fetch";
+
+const Alluser = () => {
+    const columns = [{
+        dataField: 'userID',
+        text: 'UserID',
+        sort: true
+    } , {
+        dataField: 'searchTerm',
+        text: 'Searchquery',
+        sort: true
+    },{
+        dataField: 'timestamp' ,
+        text: 'Timestamp' ,
+        sort: true
+    }];
```

Comment:
For the table headers, can we put spaces in between the words so it's easier to read? 
* `UserID` should be `User ID`
* `Searchquery` should be `Search Query`
* Timestamp is okay as is

Reply:
Fix the format.